Shell/AST.h

```diff
@@ -47,6 +47,13 @@ struct HighlightMetadata {
 struct Position {
     size_t start_offset { 0 };
     size_t end_offset { 0 };
+    struct Line {
+        size_t line_number { 0 };
+        size_t line_column { 0 };
+
+        bool operator==(const Line&) const = default;
```

Comment:
me: c++20 provides default-able comparison operators, nice
travis: heh, fool! as if I'd let you!